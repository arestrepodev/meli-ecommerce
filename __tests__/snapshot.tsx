import { render, screen } from '@testing-library/react';
import Home from '@/app/page';

describe('Home', () => {
  it('renders homepage unchanged', () => {
    const { container } = render(<Home/>)
    expect(container).toMatchSnapshot();
    expect(screen.getByTestId('search-input')).toBeInTheDocument();
  });
});
