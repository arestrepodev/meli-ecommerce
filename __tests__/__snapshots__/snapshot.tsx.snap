// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders homepage unchanged 1`] = `
<div>
  <div>
    <div
      class="container"
    >
      <header
        class="header"
      >
        <a
          href="/"
        >
          <img
            alt="logo"
            data-nimg="1"
            decoding="async"
            height="36"
            src="/_next/image?url=%2Fimages%2FLogo_ML%402x.png&w=128&q=75"
            srcset="/_next/image?url=%2Fimages%2FLogo_ML%402x.png&w=64&q=75 1x, /_next/image?url=%2Fimages%2FLogo_ML%402x.png&w=128&q=75 2x"
            style="color: transparent;"
            width="53"
          />
        </a>
        <form
          class="form"
        >
          <input
            class="searchInput"
            data-testid="search-input"
            name="search"
            placeholder="Nunca dejes de buscar"
            type="text"
          />
          <button
            class="searchButton"
            type="submit"
          >
            <img
              alt="search"
              data-nimg="1"
              decoding="async"
              height="18"
              src="/_next/image?url=%2Fimages%2Fic_Search%402x.png&w=48&q=75"
              srcset="/_next/image?url=%2Fimages%2Fic_Search%402x.png&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fic_Search%402x.png&w=48&q=75 2x"
              style="color: transparent;"
              width="18"
            />
          </button>
        </form>
      </header>
    </div>
    <div
      class="empty"
    >
      <p
        class="emptyText"
      >
        ¡Desde lo esencial hasta lo inesperado, encuentra lo que buscas aquí! 🚀
      </p>
      <p
        class="emptyText"
      >
        Usa el buscador para encontrar productos ☝️
      </p>
    </div>
  </div>
</div>
`;
